django_celery_token_bucket/bucket.py

```diff
@@ -19,9 +19,10 @@ class TokenBucket:
     """
 
     name: str
-    schedule: schedules.crontab
     amount: int
     maximum: int
+    schedule: schedules.crontab = None
```

Comment:
if we're setting None as default then we can use `Optional[schedules.crontab]` as type